docs/sources/installation/binaries.md

```diff
@@ -80,6 +80,8 @@ exhibit unexpected behaviour.
 
 ## Get the docker binary:
 
+### Latest binary for Linux
```

Comment:
@moxiegirl so what exactly have you changed? It's sort of difficult to take diff of 100 lines by eye.